wcivf/apps/elections/templates/elections/includes/_how-to-vote.html

```diff
@@ -1,61 +1,105 @@
 {% load humanize %}
-{% comment %}
+{% load i18n %}
 
+{% comment %}
 Voting system is currently one of:
 
 AMS: Additional Member System
 FPTP: First-past-the-post
 sv: Supplementary Vote
 STV: Single Transferable Vote
-PR-CL: Closed List
+PR-CL: Closed List 
+{% endcomment %}
 
-{%  endcomment %}
 <ul class="ds-details">
     <li style="list-style: none;">
         <details>
-            <summary>How you vote</summary>
+            <summary>{% trans 'How you vote' %}</summary>
             <p>
                 {% if voting_system.get_absolute_url %}
-                    This election uses <a href="{{ voting_system.get_absolute_url }}">{{ voting_system.name }}</a>.
+                    {% blocktrans trimmed with voting_system_url=voting_system.get_absolute_url %}
+                        This election uses <a href="{{ voting_system_url }}">{{ voting_system.name }}</a>.
+                    {% endblocktrans %}
                 {% else %}
-                    This election uses {{ voting_system.name }}.
+                    {% blocktrans trimmed with voting_system_name=voting_system.name %}
+                        This election uses {{ voting_system.name }}.
```

Comment:
And here

Reply:
Fixed [here](https://github.com/DemocracyClub/WhoCanIVoteFor/pull/1203/commits/c0d2f86c68959958fe350b193dbb05456c6a1b3c)